docs/resources/server_boot_device_selection.md

```diff
@@ -149,4 +149,61 @@ data "ionoscloud_image" "example" {
   location = "de/txl"
   type = "CDROM"
 }
-```
\ No newline at end of file
+```
+
+### Perform a network boot
+```hcl
+resource "ionoscloud_server_boot_device_selection" "example"{
+  datacenter_id  = ionoscloud_datacenter.example.id
+  server_id      = ionoscloud_server.example.inline_volume_ids[0]
+  # boot_device_id = data.ionoscloud_image.example.id   VM will boot in the PXE shell when boot_device_id is omitted
```

Comment:
yes, the line also contains a small note to let the user know that omission of this field will prompt the pxe boot behavior since there isn't a field dedicated to this